types/metadataV14.go

```diff
@@ -0,0 +1,397 @@
+package types
+
+import (
+	"fmt"
+	"hash"
+	"strings"
+
+	"github.com/centrifuge/go-substrate-rpc-client/v3/scale"
+	"github.com/centrifuge/go-substrate-rpc-client/v3/xxhash"
+)
+
+// Based on:
+// https://tinyurl.com/metadataV14
```

Comment:
I would point to the final github link in case the redirection doesn't work

Reply:
I agree, I only used a tinuurl because the linter failed here because the line was too long. Suggestions?

Reply:
we can add a directive to ignore the lint check like we do here - https://github.com/centrifuge/go-substrate-rpc-client/blob/master/types/event_record.go#L61